uber/models.py

```diff
@@ -1062,7 +1062,7 @@ def _misc_adjustments(self):
         if c.AT_THE_CON and self.badge_num and self.is_new:
             self.checked_in = datetime.now(UTC)
 
```

Comment:
Until now, you couldn't ever update someone's age group once it was set!  On my dev server I tried changing someone to no longer be under 18, but it just kept telling me they couldn't be a volunteer no matter what I changed their birthdate to.

I can't think of a time where the Attendee has a birthdate set and has passed all validations (so we know the birthdate is valid) but we wouldn't want to update the ``age_group``.  I've therefore changed this to just always set it.